src/products/components/ProductVariantCreatePage/ProductVariantCreatePage.tsx

```diff
@@ -90,11 +92,37 @@ const ProductVariantCreatePage: React.FC<ProductVariantCreatePageProps> = ({
             </div>
             <div>
               <Attributes
-                attributes={data.attributes}
+                title={intl.formatMessage({
```

Comment:
I think in components where we have multiple formatted messages it could make sense to move them to defineMessages so it's less stuff visually in the jsx